nomad/resource_acl_policy.go

```diff
@@ -124,6 +228,15 @@ func resourceACLPolicyRead(d *schema.ResourceData, meta interface{}) error {
 	d.Set("description", policy.Description)
 	d.Set("rules_hcl", policy.Rules)
 
+	if policy.JobACL != nil {
+		d.Set("job_acl", map[string]string{
+			"namespace": policy.JobACL.Namespace,
+			"job_id":    policy.JobACL.JobID,
+			"group":     policy.JobACL.Group,
+			"task":      policy.JobACL.Task,
+		})
```

Comment:
```suggestion
		d.Set("job_acl", []map[string]string{{
			"namespace": policy.JobACL.Namespace,
			"job_id":    policy.JobACL.JobID,
			"group":     policy.JobACL.Group,
			"task":      policy.JobACL.Task,
		}})
```

I think this is the problem. 

It's a little weird, but blocks in HCL are lists and, as the schema defines, we need a list of one element here. Calling `d.Set` with the wrong value type returns an error (which we inadvertently often ignore 😅) and so `job_acl` was never been written to state, causing the import operation to fail.

To log things while running test you can `log.Printf("[DEBUG] ...)` whatever you need and then set the env var `TF_LOG=DEBUG` while running the test.